### PR TITLE
Blog app - controllers

### PR DESCRIPTION
**Pull request by PrantoshB**
## Hello There 👋🏽 

hello

In this branch, I have:
- Created:
   - A route.
   - An action in the correct controller.
   - A view file.
as per the wireframes
- Fixed Linters Errors
- Followed Git and GitHub Flows

### Prantosh

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -3,4 +3,6 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :users, only: [:index, :show]
+  resources :posts, only: [:index, :show]
```
Comment (julie-ify) [inline config/routes.rb:7]: - [ ] The posts' routes should be nested within the users' routes. Kindly do that.

  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @PrantoshB,

Good job so far!

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (PrantoshB) [reply config/routes.rb:7]: Thanks @julie-ify for the detailed review. 🤝 
I am correcting it and submitting for a second review.

Cheers 🥂

**Review by franclobo — APPROVED** [review]
Hi @PrantoshB, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Git flow ✔️ 
- Correct routes 💯 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by PrantoshB** [comment]
Thanks @franclobo for reviewing this PR.

Have a good day 🌻

**Pull request merged**